### PR TITLE
fix activechats ordering

### PR DESCRIPTION
**Pull request by flexsurfer**
fix activechats ordering

### CODE REVIEW

**Review by vitvly — APPROVED** [review]

File: protocol/messenger.go [hunk 1]
```diff
@@ -991,6 +993,12 @@ func (m *Messenger) Init() error {
 		}
 
 		m.allChats.Store(chat.ID, chat)
+
+		if i < 60 {
```
Comment (jakubgs) [inline protocol/messenger.go:1000]: Magic number? What is it?

**Review by jakubgs — CHANGES_REQUESTED** [review]
Magic numbers are bad.

File: protocol/messenger.go (hunk 1 above)
Comment (flexsurfer) [reply protocol/messenger.go:1000]: Magic number

Comment (flexsurfer) [reply protocol/messenger.go:1000]: this function is a temporary workaround, the entire implementation is not perfect, I could add a description but...it won't explain anything :)

Comment (flexsurfer) [reply protocol/messenger.go:1000]: in will only add more questions :)

**Review by jakubgs — APPROVED** [review]
Thanks for the comment.

**Pull request merged**